modules/site-navigation/module.php

```diff
@@ -21,6 +23,15 @@ class Module extends Module_Base {
 	public function __construct() {
 		Plugin::$instance->data_manager_v2->register_controller( new Controller() );
 
+		if ( Plugin::$instance->experiments->is_feature_active( self::PAGES_PANEL_EXPERIMENT_NAME ) ) {
+			add_filter( 'elementor/editor-v2/packages/client-env', function ( $env ) {
+				$env['@elementor/editor-site-navigation'] = [
+					'is_panel_active' => true,
```

Comment:
Maybe?
If you agree you need to make sure to change it in the packages too.

```suggestion
					'is_pages_panel_active' => true,
```